# ROSbridge websocket server info
rosbridge_ip: rap-2022-ws-staff-toff.k8s.init-lab.ch #rosbridge.k8sbeta.init-lab.ch #172.17.0.1
rosbridge_port: 443 #9090
use_wss: true
# Topics being published remotely to expose locally
remote_topics:
     [
             ["/cmd_vel", "geometry_msgs/Twist"]
     ]

# Topics being published in the local roscore to expose remotely
all_local_topics: false
local_topics: [["/scan", "sensor_msgs/LaserScan"]]
# Services running remotely to expose locally
remote_services: [["/rosapi/topics", "rosapi/Topics"]]
# ["/rosapi/get_param", "rosapi/GetParam"],
# Services running locally to expose remotely
local_services: []
# Parameters to be sync, they will be polled to stay in sync
parameters: []
parameter_polling_hz: 0.03
